sklearn/tests/test_base.py

```diff
@@ -436,6 +436,7 @@ def __init__(self, attribute_pickled=5):
 
     def __getstate__(self):
```

Comment:
@jnothman I think this was supposed to test the case where users have implemented an estimator that overrides `__getstate__`